src/compute/src/lib.rs

```diff
@@ -38,37 +38,37 @@ use risingwave_common_proc_macro::OverrideConfig;
 #[derive(Parser, Clone, Debug)]
 pub struct ComputeNodeOpts {
     // TODO: rename to listen_address and separate out the port.
-    #[clap(long, default_value = "127.0.0.1:5688")]
+    #[clap(long, env, default_value = "127.0.0.1:5688")]
     pub host: String,
 
     /// The address of the compute node's meta client.
     ///
     /// Optional, we will use listen_address if not specified.
-    #[clap(long)]
+    #[clap(long, env)]
     pub client_address: Option<String>,
 
-    #[clap(long, default_value = "127.0.0.1:1222")]
+    #[clap(long, env, default_value = "127.0.0.1:1222")]
     pub prometheus_listener_addr: String,
 
-    #[clap(long, default_value = "http://127.0.0.1:5690")]
+    #[clap(long, env, default_value = "http://127.0.0.1:5690")]
     pub meta_address: String,
 
     /// Endpoint of the connector node
-    #[clap(long, env = "CONNECTOR_RPC_ENDPOINT")]
+    #[clap(long, env)]
     pub connector_rpc_endpoint: Option<String>,
 
     /// The path of `risingwave.toml` configuration file.
     ///
     /// If empty, default configuration values will be used.
-    #[clap(long, default_value = "")]
+    #[clap(long, env, default_value = "")]
     pub config_path: String,
```

Comment:
All components will share the same ENV name for `config_path`, right? Not sure whether we need to distinguish them.

Reply:
We can specify env for each process, so I think there's no need?

Reply:
I suggest adding a WARN log to prompt users we use the value from env variables. Since if there is an unexpected `CONFIG_PATH` in the user’s environment, its value will fill in.

Reply:
Let's add a `RW_` prefix, just like `POSTGRES_` in pg.

Reply:
> Not sure whether we need to distinguish them.

As long as we don't set it in some all-in-one process mode like the playground, it's fine to use the same name. 

> Let's add a RW_ prefix, just like POSTGRES_ in pg.

Good point!